<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mPlots</class>
 <widget class="QWidget" name="mPlots">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>604</width>
    <height>412</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QCustomPlot" name="pl" native="true">
      <property name="minimumSize">
       <size>
        <width>350</width>
        <height>350</height>
       </size>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QWidget" name="FunctionWidget" native="true">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <layout class="QGridLayout" name="FunctionLayout">
            <item row="4" column="0">
             <widget class="QLabel" name="XAxis">
              <property name="text">
               <string>XAxis Name</string>
              </property>
             </widget>
            </item>
            <item row="7" column="1">
             <widget class="QLineEdit" name="Left_Side">
              <property name="text">
               <string>-1</string>
              </property>
             </widget>
            </item>
            <item row="9" column="1">
             <widget class="QLineEdit" name="SplitEdit">
              <property name="text">
               <string>1000</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="Function">
              <property name="text">
               <string>x</string>
              </property>
             </widget>
            </item>
            <item row="8" column="1">
             <widget class="QLineEdit" name="Right_Side">
              <property name="text">
               <string>1</string>
              </property>
             </widget>
            </item>
            <item row="6" column="1">
             <widget class="QLineEdit" name="YAxisName">
              <property name="text">
               <string>y</string>
              </property>
             </widget>
            </item>
            <item row="7" column="0">
             <widget class="QLabel" name="LLSide">
              <property name="text">
               <string>Left Range:</string>
              </property>
             </widget>
            </item>
            <item row="9" column="0">
             <widget class="QLabel" name="Split">
              <property name="text">
               <string>Split</string>
              </property>
             </widget>
            </item>
            <item row="11" column="1">
             <widget class="QRadioButton" name="ButtonFourier">
              <property name="text">
               <string>Fourier transform</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="LFunction">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>Function:</string>
              </property>
             </widget>
            </item>
            <item row="10" column="1">
             <widget class="QRadioButton" name="ButtonFunction">
              <property name="text">
               <string>Function</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QLineEdit" name="XAxisName">
              <property name="text">
               <string>x</string>
              </property>
             </widget>
            </item>
            <item row="8" column="0">
             <widget class="QLabel" name="LRSide">
              <property name="text">
               <string>Right Range:</string>
              </property>
             </widget>
            </item>
            <item row="6" column="0">
             <widget class="QLabel" name="YAxis">
              <property name="text">
               <string>YAxis Name</string>
              </property>
             </widget>
            </item>
            <item row="12" column="1">
             <widget class="QRadioButton" name="ButtonBoth">
              <property name="text">
               <string>Both</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="ButtonOk">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Ok</string>
           </property>
           <property name="autoDefault">
            <bool>true</bool>
           </property>
           <property name="default">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ButtonClear">
           <property name="text">
            <string>Clear</string>
           </property>
           <property name="autoDefault">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Save Graph</string>
         </property>
         <property name="autoDefault">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>../../qcustomplot/qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
